src/main/views/still-working.njk

```diff
@@ -0,0 +1,27 @@
+{% extends "template.njk" %}
+{% from "govuk/components/back-link/macro.njk" import govukBackLink %}
+{% from "govuk/components/error-summary/macro.njk" import govukErrorSummary %}
+
+{% block beforeContent %}
+  {{ super() }}
+  {{ govukBackLink({
+  text: back,
+  href: "/"
+}) }}
```

Comment:
If you rebase against master. I've introduced a new generic back button.

Reply:
updated to use this